xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-api/src/main/java/org/xwiki/model/reference/EntityReference.java

```diff
@@ -42,6 +43,13 @@
  */
 public class EntityReference implements Serializable, Cloneable, Comparable<EntityReference>
 {
+    /**
+     * See {@link #getParentType()}.
+     * @since 17.0.0RC1
+     */
+    @Unstable
+    public static final String PARENT_TYPE_PARAMETER = "ParentType";
```

Comment:
The name does not seem very consistent with the "locale" parameter, we probably need a forum discussion on this choice.